components/salsa-macro-rules/src/setup_tracked_fn.rs

```diff
@@ -139,7 +139,7 @@ macro_rules! setup_tracked_fn {
                             file: file!(),
                             line: line!(),
                         };
-                        const DEBUG_NAME: &'static str = "Configuration";
+                        const DEBUG_NAME: &'static str = concat!(stringify!($fn_name), "_arguments");
```

Comment:
maybe something like?

```suggestion
                        const DEBUG_NAME: &'static str = concat!(stringify!($fn_name), "_interned_args");
```

(In general, I'm a +1 on this change.)